internal/types/document.go

```diff
@@ -277,6 +278,40 @@ func (d *Document) Remove(key string) {
 	panic(fmt.Sprintf("types.Document.Remove: key not found: %q", key))
 }
 
+// RemoveByPath removes document by path, doing nothing if the key does not exist.
+func RemoveByPath(d *Document, keys ...string) {
```

Comment:
That should be a method to match `GetByPath`

Reply:
We already have this method:

```go
func (d *Document) GetByPath(path ...string) (any, error)
```

I think it makes sense for `RemoveByPath` to be a method too:

```go
func (d *Document) RemoveByPath(path ...string)
```

Reply:
thank you, understood, made a method.